packages/eas-cli/src/build/types.ts

```diff
@@ -27,13 +27,19 @@ export interface Build {
   updatedAt: string;
   artifacts?: BuildArtifacts;
   metadata?: Partial<Metadata>;
+  error?: BuildError;
 }
 
 interface BuildArtifacts {
   buildUrl?: string;
   logsUrl: string;
 }
 
+interface BuildError {
```

Comment:
isn't it the same as ExternallError from eas-build-job ?

Reply:
Yes, it is. However, I'd prefer not to mix in that type here. The types in this file describe how the API responses look like. I think it's better to have it duplicated.